hack/install.sh

```diff
@@ -438,6 +437,10 @@ do_install() {
 			;;
 
 		fedora|centos|redhat|oraclelinux)
+			if [ "${lsb_dist}" = "redhat" ]; then
+				# we use the centos repository for both redhat and centos releases
+				lsb_dist='centos'
+			fi
```

Comment:
A little further down (line 452) we do the following.

```
if [ "$lsb_dist" = "fedora" ]
```

Should we keep them consistent? doing ```"${lsb_dist}"``` in one place and ```"$lsb_dist"``` in another makes it a little harder to read. Since they are doing the same thing, we might as well make it consistent. Is one way, better than the other? I personally don't care, which one we use.